packages/kbn-es/src/serverless_resources/project_roles/es/roles.yml

```diff
@@ -38,6 +38,9 @@ developer:
 
 # admin role defined in elasticsearch controller
 admin:
+  metadata:
+    _public: true
+    _reserved: true
```

Comment:
@slobodanadamovic Was this missed? OBLT and Security serverless roles have metadata for the admin role, but the search solution did not. Feel free to remove this if it shouldn't be here.